src/Scrollbar.js

```diff
@@ -515,6 +515,6 @@ var Scrollbar = createReactClass({
 
 Scrollbar.KEYBOARD_SCROLL_AMOUNT = KEYBOARD_SCROLL_AMOUNT;
 Scrollbar.SIZE = parseInt(cssVar('scrollbar-size'), 10);
-Scrollbar.OFFSET = FACE_MARGIN/2 + 1;
+Scrollbar.OFFSET = 1;
```

Comment:
@miskreant the scrollbar spacing seemed incorrect, so I made this logical change.  Could you double check this?

Reply:
Looks fine to me, since w/o this change there's an extra border visible.